### ISSUE TITLE
URLs for HDF5-1.8.18-intel-2016b are no longer correct.

### ISSUE THREAD

**Issue by pol** [easybuilders/easybuild-easyconfigs#4882]
Tried to install with `[easybuild@hyalite ~]$ eb HDF5-1.8.18-intel-2016b.eb --robot`, failed saying it could not find hdf5-1.8.18.tar.gz:

```
EasyBuild crashed with an error (at ?:124 in __init__): Couldn't find file hdf5-1.8.18.tar.gz anywhere, and downloading it didn't work either... Paths attempted...<SNIP>
https://support.hdfgroup.org/ftp/HDF5/current18/src/hdf5-1.8.18.tar.gz, https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(versions)/src/hdf5-1.8.18.tar.gz
```

The error is correct, neither of those URLs work.   The current18 folder now only contains hdf5-1.8.19.tar.gz, and the specific version URL should actually be (I think): https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-$(versions)/src/

I solved this problem by manually downloading the file into the sources directory:

```
[easybuild@hyalite ~]$ cd  /cm/shared/hyalite/apps/eb/sources/h/HDF5/
[easybuild@hyalite HDF5]$ wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.18/src/hdf5-1.8.18.tar.gz
```

**Comment by boegel** [comment]
@pol This was fixed in #4749; if you are using EasyBuild v3.3.1 (released yesterday), you shouldn't be seeing this problem anymore.

**Comment by pol** [comment]
I'll reopen if it fails post-update (doubtful).  Thanks for the response.
